src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -41,12 +41,12 @@
              error (style/create-server-error-message error)
              :else [comps/Spinner {:text "Loading details..."}])))
    :component-did-mount
-   (fn [{:keys [this]}]
-     (this :load-agora-method))
+   (fn [{:keys [this props state]}]
+     (react/call :load-agora-method this props state))
    :load-agora-method
-   (fn [{:keys [props state]}]
+   (fn [{:keys [state]} method-ref]
      (endpoints/call-ajax-orch
-      {:endpoint (endpoints/get-agora-method (:namespace props) (:name props) (:snapshotId props))
+      {:endpoint (endpoints/get-agora-method (:namespace method-ref) (:name method-ref) (:snapshotId method-ref))
        :headers utils/content-type=json
```

Comment:
I changed this back to the way it was because it introduced a bug!
If you attempted to edit the method config and had the WDL expanded and then tried to change the snapshot id, the WDL would remain the same. Restoring this got rid of the bug